### ISSUE TITLE
HorizontalStackLayout to use full width

### ISSUE THREAD

**Issue by williambuchanan2** [dotnet/maui#14177]
### Description

With StackLayout set in Horizontal orientation, you can add controls and they take up the full width of the page (which is a frequent requirement).

It seems this isn't possible with HorizontalStackLayout, although we are being advised to use it instead of StackLayout.

Could there be an option in HorizontalStackLayout to make it behave like StackLayout?

### Public API Changes

No idea

### Intended Use-Case

I would say 99% of the time I expect the full width to be used.

**Comment by SarthakB26** [comment]
@williambuchanan2  use StackLayout with orientation horizontal instead of HorizontalStackLayout, FillAndExpands work there

**Comment by PureWeen** [comment]
@williambuchanan2 here's a wiki describing the changes with layouts in MAUI
https://github.com/dotnet/maui/wiki/Xamarin.Forms----MAUI.Controls-Layout-Differences

**Comment by williambuchanan2** [comment]
> 

Yes, that is what I am doing. The point I am raising is not related to that.

**Comment by hartez** [comment]
There's nothing which requires you to use the HorizontalStackLayout. If both StackLayout and HorizontalStackLayout meet your requirements, then HorizontalStackLayout is a _slightly_ more performant option. If the HorizontalStackLayout does not meet your requirements, then you should use a layout which does.

If you're looking for a control which will automatically span the exact width of the the container, then your options include Grid, FlexLayout, and StackLayout (using the obsolete but currently still supported "AndExpand" options).
